### DISCUSSION TITLE
Feature Request: ROCm Support for AMD Ryzen 9 7940HS with Radeon 780M Graphics

### DISCUSSION THREAD

**Discussion by pearsonc** [ROCm/TheRock#8053] [Ideas]
Hi ROCm Development Team,

I hope this message finds you well. I am writing to request the development of ROCm (Radeon Open Compute) support for the AMD Ryzen 9 7940HS processor paired with Radeon 780M graphics.

The AMD Ryzen 9 7940HS is increasingly becoming a popular choice among both professional and enthusiast communities due to its impressive performance and energy efficiency. With its powerful integrated Radeon 780M graphics, this chip is making its way into a variety of setups, particularly in home labs, small-scale development environments, and DIY computing projects.

However, the lack of ROCm support for this processor is a significant drawback for many users. The home lab community, which often relies on AMD hardware for its balance of cost, performance, and energy efficiency, is currently feeling excluded. This lack of support is leading to considerable frustration and regret among users who have invested in this otherwise excellent hardware.

Integrating ROCm support for the Ryzen 9 7940HS would not only enhance the capabilities of these systems but also expand the reach and adoption of AMD’s hardware in various computing environments. The home lab community, in particular, stands to benefit greatly from this support, enabling them to leverage ROCm’s powerful computing capabilities for a range of applications, from machine learning and AI development to advanced data analytics and scientific computing.

Moreover, extending ROCm support to this processor will demonstrate AMD’s commitment to providing comprehensive support across its product range, thereby strengthening customer loyalty and satisfaction. Given the growing popularity of the Ryzen 9 7940HS, I believe that this enhancement aligns well with AMD’s strategic goals and market demands.

Thank you for considering this request. I am confident that this addition would be highly appreciated by a broad segment of your user base and would significantly enhance the usability and appeal of AMD’s hardware in diverse computing environments.

Best regards,

Chris Pearson

**Comment by pearsonc** [comment]
I have opened an actual feature request here: https://github.com/ROCm/ROCm/issues/3398

Please go and show support for this if you want to push for its development.

**Comment by AlexHe99** [comment]
Hello, I have committed a pull request  https://github.com/ollama/ollama/pull/5426 to guide how to enable ROCm for AMD iGPU780M.

**Comment by dreirund** [comment]
\+1

**Comment by trixirt** [comment]
I have built experimental support for much of the stack in Fedora rawhide.
Unfortunately I have not had time to test this.
To use do
> module load rocm/gfx1103

This sets up the environment to use rocBLAS and friends build for gfx1103/780M.
 When you run into problems, file a bz with Fedora.

**Comment by MGAndreasen** [comment]
how about AMD HIP SDK, when will we see support for all zen apus on this list, and not just the resently released AI apus?

**Comment by trixirt** [comment]
The experimental support for 780M I mentioned above was turned off by default in Fedora 42 along with several other apus.

**Comment by harkgill-amd** [comment]
Apologies for the lack of update here but since this was originally reported, ROCm has extended support to gfx1103/Radeon 780M. You can find the latest release with install steps over at https://rocm.docs.amd.com/en/latest/install/rocm.
